### PR TITLE
Added personal keymap for keebio/iris/rev6

### PR DESCRIPTION
**Pull request by adamgate**
## Description

Added my keymap for the keebio iris rev 6. Ensured that the changes are in line with the QMK PR keymap requirements.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [X] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [X] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [X] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keebio/iris/keymaps/adamgate/keymap.c [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+  THE SOFTWARE.
+*/
+
+#include QMK_KEYBOARD_H
+
+#define _QWERTY 0
+#define _LOWER  1
+#define _GAMING 2
+#define _RAISE  3
```
Comment (drashna) [inline keyboards/keebio/iris/keymaps/adamgate/keymap.c:28]:
```suggestion
enum custom_layers {
  _QWERTY,
  _LOWER,
  _GAMING,
  _RAISE,
};
```

File: keyboards/keebio/iris/keymaps/adamgate/keymap.c [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+        layer_on(_RAISE);
+      } else {
+        layer_off(_RAISE);
+      }
+      return false;
+      break;
+      
+  }
+  return true;
+}
```
Comment (drashna) [inline keyboards/keebio/iris/keymaps/adamgate/keymap.c:127]: These all can be removed.  The layer keycodes can be replaced with `MO(x)`, for instance.
```suggestion
```

File: keyboards/keebio/iris/keymaps/adamgate/rules.mk [hunk 3]
```diff
@@ -0,0 +1,7 @@
+SPLIT_KEYBOARD = yes
```
Comment (drashna) [inline keyboards/keebio/iris/keymaps/adamgate/rules.mk:1]: Shouldn't need this.
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by adamgate** [comment]
I made the requested changes on my fork, but I don't know how to ensure that they end up on this PR. Github keeps suggesting that I should make a new PR instead.

**Comment by adamgate** [comment]
Made the requested changes.

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**